aria_data_loaders/aria_data_utils/detector_wrappers/april_tag_detector.py

```diff
@@ -134,17 +133,18 @@ def get_outputs(
         """
         # Decorate image with text for visualization
         img = decorate_img_with_text(
-            img=img_frame,
+            img=img_frame.get("image"),
             frame_name="device",
             position=device_T_marker.translation(),
         )
+        timestamp = img_frame.get("timestamp")
         print(
```

Comment:
I'd prefer rospy.logdebug every where you have debugging prints currently.

Reply:
+1

Reply:
addressed